tendrl/node_agent/flows/execute_command.py

```diff
@@ -0,0 +1,13 @@
+from tendrl.node_agent.atoms.generic.command import Command
+
+
+class ExecuteCommand(object):
+    def __init__(self, api_job):
+        super(ExecuteCommand, self).__init__()
+        self.api_job = api_job
+        self.atom = Command
+
+    def start(self):
+        attributes = self.api_job['attributes']
+        result, err = self.atom().start(attributes)
```

Comment:
All the key-value pairs of the attributes dictionary act as Key word args for the runner function. For example, in case of package installation atom the attributes dict will be something like
{
"name" : "gluster",
"state": "installed"
}
So the whole dictionary is needed by atom.